utils/setup/gen_ld_preload.py

```diff
@@ -11,12 +11,30 @@
 script_dir = os.path.dirname(os.path.realpath(__file__))
 ld_preload = list()
 
-for path in Path("/").rglob("libgomp*"):
-    if ".so" in path.name:
+try:
+    for path in Path("/").rglob("libgomp*"):
+        if ".so" in path.name:
+            ld_preload.append(str(path))
+except OSError as e:
+    print(e)
+    for path in Path("/lib").rglob("libgomp*"):
```

Comment:
What's the reason behind this strange "error-handling"?

Reply:
when I search from root when run in Jenkins this is an error which python throws. However when I run from anywhere below like "/lib" it works as usual.

Reply:
Jenkins isn't allowed to run from /root directly on agent. Try use path like `/root/miniforge3/envs` maybe it will works better.

Reply:
I see, I will change it for "/root" however other packages are located in "/lib"